cni/deployments/kubernetes/install/test/install_cni.go

```diff
@@ -249,6 +264,17 @@ func doTest(testNum int, wd, preConfFile, resultFileName, expectedOutputFile,
 	}()
 	time.Sleep(10 * time.Second)
 
+	if delayedConfFile != "NONE" {
+		var destFilenm string
+		if preConfFile != "NONE" {
+			destFilenm = preConfFile
+		} else {
+			destFilenm = delayedConfFile
+		}
+		cp(delayedConfFile, tempCNIConfDir+"/"+destFilenm, t)
+		time.Sleep(10 * time.Second)
```

Comment:
This is too long. Unit tests should be sub-second.
Replace this with a loop that checks every second with a maximum number of retries (timeout).